bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -269,7 +269,17 @@ class PlotCanvasView extends Renderer.View
       @canvas_view.set_dims([info.dimensions.width, info.dimensions.height])
 
   reset_dimensions: () ->
-    @canvas_view.set_dims([@canvas.initial_width, @canvas.initial_height])
+    @model.plot.width = @canvas.initial_width
+    @model.plot.height = @canvas.initial_height
+    @model.document.resize()
```

Comment:
I have no idea. I changed this so that reset_dimensions would work. It didn't have a pause() in there before, so I didn't add one. We could.